selfdrive/camerad/main.cc

```diff
@@ -1062,6 +1167,32 @@ void init_buffers(VisionState *s) {
     assert(err == 0);
   }
 
+  s->prg_debayer_rear = build_debayer_program(s, s->cameras.rear.ci.frame_width, s->cameras.rear.ci.frame_height,
+                                                   s->cameras.rear.ci.frame_stride,
+                                                 s->rgb_width, s->rgb_height, s->rgb_stride,
+                                                 s->cameras.rear.ci.bayer_flip, s->cameras.rear.ci.hdr);
+  s->krnl_debayer_rear = clCreateKernel(s->prg_debayer_rear, "debayer10", &err);
```

Comment:
Where was this created initially? Wasn't this already there somewhere?

Reply:
oops that's a dupl from my copy and paste :|